src/components/Calculator.js

```diff
@@ -1,30 +1,67 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
+
 // eslint-disable-next-line react/prefer-stateless-function
+/* eslint-disable */
 class Calculator extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state={
+      total : null,
+      next:null,
+      operation:null
+    }
+    this.getValue=this.getValue.bind(this);
+  }
+
+  getValue=(e)=>{
+    const buttonName  =e.target.value;
+    this.setState((prevValue)=>calculate({
+      ...prevValue,
+    },buttonName ));
+    
+  }
+
+  result = () => {
+    const { total, operation, next } = this.state;
+    return (
+      <>
+        {total}
+        {' '}
+        {operation}
+        {' '}
+        {next}
+      </>
+    );
+  }
+  
+  
   render() {
     return (
       <div className="container">
-        <div className="resualt"><h5>0</h5></div>
+        <div className="resualt"><h5>{this.result()}</h5></div>
         <div className="content">
-          <div className="item1"><p>Ac</p></div>
-          <div className="item2"><p>+/-</p></div>
-          <div className="item3"><p>%</p></div>
-          <div className="calculation"><p>*</p></div>
-          <div className="item1"><p>7</p></div>
-          <div className="item1"><p>8</p></div>
-          <div className="item2"><p>9</p></div>
-          <div className="calculation"><p>x</p></div>
-          <div className="item1"><p>4</p></div>
-          <div className="item1"><p>5</p></div>
-          <div className="item2"><p>6</p></div>
-          <div className="calculation"><p>-</p></div>
-          <div className="item1"><p>1</p></div>
-          <div className="item1"><p>2</p></div>
-          <div className="item2"><p>3</p></div>
-          <div className="calculation"><p>+</p></div>
-          <div className="zero"><p>0</p></div>
-          <div className="item1"><p>.</p></div>
-          <div className="calculation"><p>=</p></div>
+          <button value='Ac' onClick={this.getValue} type='submit'>Ac</button>
```

Comment:
- [ ] As I notice your Ac button is not working. you can fix it by providing value with `AC` not Ac because in logic/calculate.js your buttons name is AC. Kindly fix it

Reply:
@roshan-bajgain !
Thank you for your feedback. I'm going to fix this error. :+1: